rtrlib/rtr_mgr.c

```diff
@@ -293,7 +293,7 @@ int rtr_mgr_init(struct rtr_mgr_config **config_out,
 	struct pfx_table *pfxt = NULL;
 	struct spki_table *spki_table = NULL;
 	struct rtr_mgr_config *config = NULL;
-	*config_out = NULL;
```

Comment:
I think we should keep setting *config_out = NULL, because if groups_len = 0 the value of *config_out is now undefined, but should be NULL according the doxygen comment.